latest/src/app/documentation/demos/datagrid/selection/selection.html

```diff
@@ -97,3 +97,8 @@ <h2>Selection</h2>
     selected, you can simply write:
 </p>
 <clr-code-snippet [clrCode]="singleRowExample" clrLanguage="html"></clr-code-snippet>
+
+<p>
+    If you need to listen to when the selection changes, you can bind to the <code class="clr-code">(clrDgSelectedChange)</code> event:
```

Comment:
Since this is not Clarity but just Angular's convention, could you please add just a simple link to the corresponding documentation: https://angular.io/guide/template-syntax#two-way-binding--- ?